
<!-- saved from url=(0121)https://bug698264.bugzilla.mozilla.org/attachment.cgi?id=570800&action=diff&collapsed=&context=patch&format=raw&headers=1 -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="chrome-extension://dbmjjjonelodfeckmpfglmffhngdplal/page.js" type="text/javascript"></script></head><body><pre style="word-wrap: break-word; white-space: pre-wrap;">--- a/xpcom/build/nsXPCOMPrivate.h	Sat Oct 29 09:52:45 2011 -0700
+++ a/xpcom/build/nsXPCOMPrivate.h	Mon Oct 31 20:43:27 2011 +0100
@@ -257,7 +257,7 @@ 
 #else // Unix
 #include &lt;limits.h&gt; // for PATH_MAX
 
-#define XPCOM_DLL "libxpcom"MOZ_DLL_SUFFIX
+#define XPCOM_DLL "libxpcom" MOZ_DLL_SUFFIX
 
 // you have to love apple..
 #ifdef XP_MACOSX  
@@ -266,7 +266,7 @@ 
 #define XUL_DLL            "XUL"
 #else
 #define XPCOM_SEARCH_KEY  "LD_LIBRARY_PATH"
-#define XUL_DLL   "libxul"MOZ_DLL_SUFFIX
+#define XUL_DLL   "libxul" MOZ_DLL_SUFFIX
 #endif
 
 #define GRE_CONF_NAME ".gre.config"
</pre></body></html>